examples/browser/webpack.config.js

```diff
@@ -7,6 +7,10 @@ module.exports = {
     path: path.resolve(__dirname, "dist"),
     filename: "bootstrap.js",
   },
+  experiments: {
+    asyncWebAssembly: true,
+    syncWebAssembly: true,
```

Comment:
```suggestion
```
This can be removed and we can just leave the `asyncWebAssembly` to support the new way of importing WebAssembly modules